### PR TITLE
Always exit inspector on any error with an error

### LINKED ISSUE #186: Installation not terminating on validation error
With errors during `run pre-flight checks using Kismatic Inspector` the installer doesn't terminate and continues with the install. 

**OS**: CentOS7 on GCE

```
Validating==========================================================================
Reading installation plan file "kismatic-cluster.yaml"                          [OK]
Validating installation plan file                                               [OK]
1/3....  Configure Cluster Prerequisites
- Running task: Install python 2.7
  node1                                                                         [OK]
  node2                                                                         [OK]
- Running task: create a temporary download directory
  node1                                                                         [OK]
  node2                                                                         [OK]
1/3....  Configure Cluster Prerequisites Finished
2/3....  Update Hosts File
- Running task:
  node1                                                                         [OK]
  node2                                                                         [OK]
- Running task: remove this hostname from hosts file
  node1                                                                         [OK]
  node2                                                                         [OK]
- Running task: add nodes to hosts file
  node1                                                                         [OK]
  node2                                                                         [OK]
  node1                                                                         [OK]
  node1                                                                         [OK]
  node2                                                                         [OK]
  node2                                                                         [OK]
2/3....  Update Hosts File Finished
3/3....  Run Cluster Pre-Flight Checks
- Running task:
  node1                                                                         [OK]
  node2                                                                         [OK]
- Running task: verify systemd
  node2                                                                         [SKIPPED]
  node1                                                                         [SKIPPED]
- Running task: copy Kismatic Inspector to node
  node1                                                                         [OK]
  node2                                                                         [OK]
- Running task: copy kismatic-inspector.service to remote
  node1                                                                         [OK]
  node2                                                                         [OK]
- Running task: start kismatic-inspector service
  node1                                                                         [OK]
  node2                                                                         [OK]
- Running task: run pre-flight checks using Kismatic Inspector
  node2                                                                         [ERROR]
---- STDERR ----
Error: error running inspector against remote node: error posting request to server: Post http://104.197.142.192:8888/execute: dial tcp 104.197.142.192:8888: i/o timeout
---------------
  node1                                                                         [ERROR]
---- STDERR ----
Error: error running inspector against remote node: error posting request to server: Post http://130.211.120.125:8888/execute: dial tcp 130.211.120.125:8888: i/o timeout
---------------
- Running task: stop kismatic-inspector service
  node2                                                                         [OK]
  node1                                                                         [OK]
- Running task: stop kismatic-inspector service
  node1                                                                         [OK]
  node2                                                                         [OK]
- Running task: preflight : reload services
  node1                                                                         [OK]
  node2                                                                         [OK]
3/3....  Run Cluster Pre-Flight Checks Finished

Configuring Certificates============================================================
Found key and certificate for node "node1"                                      [OK]
Found key and certificate for node "node2"                                      [OK]
Found key and certificate for service accounts                                  [OK]
Found key and certificate for user "admin"                                      [OK]
Cluster certificates can be found in the "generated" directory                  [OK]
```

### PR DESCRIPTION
**Pull request by dkoshkin**
Fixes #186

### CODE REVIEW

File: ansible/roles/preflight/tasks/main.yaml [hunk 1]
```diff
@@ -18,23 +18,30 @@
...
+        service:
+          name: kismatic-inspector.service
+          state: stopped
     always:
       - name: stop kismatic-inspector service
         service:
           name: kismatic-inspector.service
           state: stopped
+
+  - assert:
```
Comment (alexbrand) [inline ansible/roles/preflight/tasks/main.yaml]: What error is shown in this case? I think we have used a similar model in other places, but with the "fail... when" task, which can be given a name and message.

**Comment by alexbrand** [comment]
Do we get the output from the inspector in the case that there is a failure?

**Comment by dkoshkin** [comment]
Yes that comes from the task that runs the command.

**Comment by alexbrand** [comment]
LGTM

**Pull request merged**